app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,79 @@
+class ReviewsController < ApplicationController
+  before_action :check_user
+  before_action :store_location, only: %i(edit)
+  before_action :find_booking, only: %i(new)
+  before_action :check_owner, only: %i(destroy edit update)
+  def new
+    @review = Review.new(booking: @booking)
```

Comment:
```suggestion
    @review = @booking.review.new 
```
Như này đc không nhỉ

Reply:
Em chưa thêm  accepts_nested_attributes_for nên chưa được anh để em thêm ạ

Reply:
em thử ko dc anh oii